flutter/lib/app_constants.dart

```diff
@@ -37,11 +39,11 @@ class AppColors {
           const Color(0xFF0B4A7F),
         ]
       : [
-          Colors.brown.shade400,
```

Comment:
@freedomtan Can you please review this PR again? There was a merge conflict so I need to resolve it.
Regarding the color, here is the screenshot, the brown color is now a bit darker.

The reason is that I now use the system default `MaterialColor` brown for the app bar and it's a bit darker.

BEFORE:
![Simulator Screen Shot - iPhone 14 Pro - 2023-03-21 at 12 50 55 Medium](https://user-images.githubusercontent.com/85728587/226528610-76c3d03b-141e-4ab5-bedb-84b455e61325.jpeg)

AFTER:
![Simulator Screen Shot - iPhone 14 Pro - 2023-03-21 at 12 47 05 Medium](https://user-images.githubusercontent.com/85728587/226528618-d4690f43-d6a9-4ffb-ba64-fbe08041bee9.jpeg)